src/commands/admin/appcommands.ts

```diff
@@ -95,10 +100,47 @@ export default class AppCommandsCommand implements BaseCommand {
                           },
                       ];
 
-            for (const command of commandsToModify) {
+            for (const commandObj of commandsToModify) {
+                const commandName = commandObj[0];
+                const command = commandObj[1];
                 if (command.slashCommands) {
-                    const commands = command.slashCommands();
+                    const commands =
+                        command.slashCommands() as Array<Eris.ChatInputApplicationCommandStructure>;
+
                     for (const cmd of commands) {
+                        cmd.name =
+                            cmd.name ??
+                            i18n.translate(
+                                LocaleType.EN,
+                                `command.${commandName}.help.name`
```

Comment:
Can we throw an exception if the 'name' translation key is missing, but we're expecting it to exist? Or a test